src/libs/actions/BankAccounts.js

```diff
@@ -906,6 +906,22 @@ function validateRoutingNumber(number) {
     return false;
 }
 
+function getOAuthReceivedRedirectURI() {
+    if (!window.location) {
+        return null;
+    }
+
+    let receivedRedirectURI = window.location.href;
+    const receivedRedirectSearchParams = (new URL(receivedRedirectURI)).searchParams;
+    const oauthStateID = receivedRedirectSearchParams.get('oauth_state_id');
+
+    // If there's no stateID passed, then return user to start of Plaid flow by setting the redirectURI to null
```

Comment:
Hows does setting this to `null` "return user to start of Plaid flow"?

Reply:
If we pass `null` for the `redirectURI` then we clear the existing token and fetch a new one here https://github.com/Expensify/App/blob/6842a4722847a88d289803b44e906c4c6428f8a1/src/components/AddPlaidBankAccount.js#L113

Reply:
Ok I would probably move that comment somewhere else. This method is just pulling some data out of a URL and could be used in different ways. Maybe instead we can say something about why the url is valid only when it has a `stateID`?